jetty-server/src/main/java/org/eclipse/jetty/server/ServerConnector.java

```diff
@@ -336,13 +336,6 @@ protected ServerSocketChannel openAcceptChannel() throws IOException
 
         return serverChannel;
     }
-    
-    @Override
-    public Future<Void> shutdown()
-    {
-        // shutdown all the connections
-        return super.shutdown();
-    }
 
```

Comment:
How come this got removed by this PR?

Reply:
It's a strange change, not related to the scope of the issue, but it's ultimately harmless.

It just calls `super.shutdown();` 

[AbstractConnector.shutdown()](https://github.com/eclipse/jetty.project/blob/f046bdc2c61584462949ea9aee10b32a43bb74d6/jetty-server/src/main/java/org/eclipse/jetty/server/AbstractNetworkConnector.java#L102-L107) is where the work is anyway.

Reply:
because it is a noop.  I can put it back if you really want.... (yes I'm breaking my own rules for putting in unrelated changes).